src/lib/y2partitioner/widgets/configure.rb

```diff
@@ -0,0 +1,276 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/widgets/reprobe"
+
+Yast.import "Stage"
+Yast.import "Popup"
+Yast.import "Arch"
+Yast.import "PackageCallbacks"
+Yast.import "PackageSystem"
+
+module Y2Partitioner
+  module Widgets
+    # "Configure" menu button used to run the corresponding YaST clients to
+    # activate several storage technologies
+    #
+    # Most of the behavior of this widget is a direct translation of the old
+    # Yast::PartitioningEpAllInclude from yast-storage
+    class Configure < CWM::CustomWidget
+      include Reprobe
+
+      # Constructor
+      def initialize
+        textdomain "storage"
+        super
+      end
+
+      # Content of the widget, a menu button with the list of available
+      # configuration clients or an empty widget if no client is available
+      #
+      # @macro seeCustomWidget
+      #
+      # @return [Yast::WidgetTerm]
+      def contents
+        @contents ||= actions.empty? ? Empty() : MenuButton(opt, label, items)
+      end
+
+      # Event handler for the configuration menu
+      #
+      # @param event [Hash] UI event
+      # @return [:reprobe, nil] :reprobe when some configuration client was
+      #   executed; nil otherwise.
+      def handle(event)
+        action = Action.find(event["ID"])
+
+        return nil unless action
+        return nil unless warning_accepted?(action) && availability_ensured?(action)
+
+        Yast::WFM.call(action.client)
+        reprobe
+        :reprobe
+      end
+
+    private
+
+      # @return [Array<Yast::WidgetTerm>]
+      def items
+        actions.map do |action|
+          Item(Id(action.id), Yast::Term.new(:icon, action.icon), action.label)
+        end
+      end
+
+      # @return [Yast::WidgetTerm]
```

Comment:
please try to be consistent with CWM itself which also have opt and here it very confuse me why it have different syntex. See please https://github.com/yast/yast-yast2/blob/33d7484d4827421d11da27b23d6317ef7d2d5b95/library/cwm/src/lib/cwm/abstract_widget.rb#L92

so above in widget should be `Opt(*opt), label, items`